src/modules/exam/dto/create-exam.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { IsBoolean, IsNotEmpty, IsString, IsUUID } from 'class-validator';
+
+export class CreateExamDto {
+  @IsString()
+  @IsNotEmpty()
+  name: string;
+
+  @IsString()
+  @IsNotEmpty()
+  description: string;
+
+  @IsString()
+  @IsNotEmpty()
+  duration: string;
+
+  @IsBoolean()
+  @IsNotEmpty()
+  isPublic: boolean;
+
+  @IsUUID()
+  @IsNotEmpty()
+  teacherId: string;

```

Comment:
Do we allow creating exams for another user, if not I think we shouldn't take `teacherId` here...use current user instead

Reply:
When function  Authen and Author functions is complete, I will use current user instead teacherId